src/components/PdfReaderComponent.js

```diff
@@ -96,6 +99,26 @@ class PdfReaderComponent {
       this.components.pdfContainer.innerHTML = "";
       this.loadPdf(pdf);
     });
+
+    EventHandlerService.subscribe(
+      PDFLEvents.onKeyboardKeyDown,
+      (functionalKeys, key) => {
+        if (!functionalKeys.ctrl) {
+          return;
+        }
+        if (key === "u") {
+          this.#onNewFile();
+        }
+      }
+    );
+  };
+
+  /**
+   * Cretes event triggered when application view changed from reader view to input view.
```

Comment:
I know you did not write this, but this jsdoc needs to be written differently.

Reply:
This shouldn't  be here at all, removed now